src/auth/Crypto.cc

```diff
@@ -131,6 +131,20 @@ std::size_t CryptoKeyHandler::decrypt(
   return todo_len;
 }
 
+sha256_digest_t CryptoKeyHandler::hmac_sha256(
+  const ceph::bufferlist& in) const
+{
+  ceph::crypto::HMACSHA256 hmac((const unsigned char*)secret.c_str(), secret.length());
```

Comment:
IIUC `ceph::crypto::HMACSHA256` still uses NSS inside. We should consider switching it to OpenSSL.